biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/embl/EmblId.java

```diff
@@ -0,0 +1,108 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+/**
+ *
+ * This class contains the processed data of embl file
+ * Primary accession number
+ * Sequence version number
+ * Topology: 'circular' or 'linear'
+ * Molecule type
+ * Data class
+ * Taxonomic division
+ * Sequence length
+ * @since 5.0.0
+ * @author Noor Aldeen Al Mbaidin
+ */
+public class EmblId {
```

Comment:
If all the fields were marked final and only settable in the constructor, this class could be marked `@Immutable`

Reply:
done.